kusari/cmd/repo_scan.go

```diff
@@ -4,33 +4,43 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/kusaridev/kusari-cli/pkg/repo"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var (
-	platformUrl string
-	wait        bool
+	platformUrl  string
+	wait         bool
+	outputFormat string
 )
 
 func init() {
 	scancmd.Flags().StringVarP(&platformUrl, "platform-url", "", "https://platform.api.us.kusari.cloud/", "platform url")
 	scancmd.Flags().BoolVarP(&wait, "wait", "w", true, "wait for results")
+	scancmd.Flags().StringVarP(&outputFormat, "output-format", "", "markdown", "output format (markdown or SARIF)")
 
 	// Bind flags to viper
 	mustBindPFlag("platform-url", scancmd.Flags().Lookup("platform-url"))
 	mustBindPFlag("wait", scancmd.Flags().Lookup("wait"))
+	mustBindPFlag("output-format", scancmd.Flags().Lookup("output-format"))
 }
 
 func scan() *cobra.Command {
 	scancmd.RunE = func(cmd *cobra.Command, args []string) error {
 		cmd.SilenceUsage = true
 
+		// Validate output format
+		if outputFormat != "markdown" && outputFormat != "SARIF" {
```

Comment:
My only question is why lowercase one and not the other... Markdown is capitalized, normally: https://en.wikipedia.org/wiki/Markdown
I'd probably just lowercase both of them, myself.

Zizmor uses lowercase for SARIF: https://docs.zizmor.sh/usage/#sarif